BackEnd/src/main/java/com/exadel/carpoolfree/controller/MessageController.java

```diff
@@ -0,0 +1,46 @@
+package com.exadel.carpoolfree.controller;
+
+
+import com.exadel.carpoolfree.model.Message;
+import com.exadel.carpoolfree.repository.MessageRepository;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("api/chat")
+public class MessageController {
+
+    private final MessageRepository messageRepository;
+
+    public MessageController(MessageRepository messageRepository) {
+        this.messageRepository = messageRepository;
+    }
+
+    @GetMapping("/{id}")
+    public Message getById(final @PathVariable Long id) {
+        return messageRepository.findById(id).get();
+    }
+
+    @GetMapping()
+    public List<Message> getAllMessages() {
```

Comment:
Are we going to need this on UI? Is it sfe to delete this?

Reply:
Method was deleted.